crates/wash-cli/src/down/mod.rs

```diff
@@ -59,8 +73,13 @@ pub struct DownCommand {
     #[clap(long = "host-id")]
     pub host_id: Option<ServerId>,
 
+    /// Shutdown all hosts running locally if launched with --multi-local
     #[clap(long = "all")]
     pub all: bool,
+
+    /// Purge NATS Jetstream storage and streams that persist when wasmCloud is stopped
+    #[clap(long = "purge", alias = "flush")]
+    pub purge: PurgeJetstream,
```

Comment:
Can we name this in a way that reflects it's purging NATS/Jetstream in particular?

Reply:
Do you think like, `--purge-nats` or `--purge-jetstream`? From those I kinda do prefer `--purge` but I'm not that hard-convinced

Reply:
`--purge-jetstream` is probably pretty reasonable? Or maybe something like `--purge-state`, because that's what you're really purging right?

Reply:
Maybe `--purge-nats-state`? Because you're really purging state, but only the stuff in NATS -- not like you're cleaning out the filesystem

Reply:
I think `purge-jetstream` feels fine here, I know in general we could have a `--purge` if we also added support for some of the other drain parameters like cache or removing the `~/.wash` directory

Reply:
Perfect! Resolving for :rocket: